binder/views.py

```diff
@@ -1614,6 +1645,97 @@ def _store_field(self, obj, field, value, request, pk=None):
 								}
 							})
 					setattr(obj, f.attname, value)
+
+				elif isinstance(f, models.FileField):
+					if not isinstance(value, File) and value is not None:
+						raise BinderFieldTypeError(self.model.__name__, field)
+
+					if value is not None:
+						if value.size > self.max_upload_size * 10**6:
+							raise BinderFileSizeExceeded(self.max_upload_size)
+
+						if getattr(f, 'allowed_extensions', None) is not None:
+							extension = None if '.' not in value.name else value.name.split('.')[-1]
```

Comment:
We should use [os.path.splitext](https://docs.python.org/3/library/os.path.html#os.path.splitext), instead of reinventing the wheel all the time.

Reply:
If this was newly written code I agree. However in this case it was just moved from dispatch_file_field to store_field, in which case I think its more important to remain 100% backwards compatible.

Reply:
I disagree, since the only reason I can think that it is not 100% backwards compatible, is if the code gives a wrong extension. That would be a security issue, and a bug that needs to be fixed